packages/flutter_tools/lib/src/android/adb.dart

```diff
@@ -204,7 +204,7 @@ class AdbDevice {
   /// Device model; can be null. `XT1045`, `Nexus_7`
   String get modelID => _info['model'];
 
-  set modelID(String value) {
+  void set modelID(String value) {
```

Comment:
Isn't the `void` implied, or best practice? It would be nice if the lint ignored setters that didn't have return types.